common/src/main/java/com/microsoft/identity/common/internal/authorities/AzureActiveDirectoryAuthority.java

```diff
@@ -183,4 +185,21 @@ public AzureActiveDirectoryAudience getAudience() {
         return mAudience;
     }
 
+    /**
+     * Checks if current authority cloud has same preferred network host name as the passed in authority cloud
+     *
+     * @param authorityToCheck authority to check against
+     * @return true if preferred network host name matches for both authorities, otherwise false
+     */
+    @WorkerThread
+    public synchronized boolean isSamePreferredNetworkHostAsAuthority(@NonNull final AzureActiveDirectoryAuthority authorityToCheck) throws IOException {
+        if (!AzureActiveDirectory.isInitialized()) {
```

Comment:
I don't love this API, but sure.  We have two function calls where one might do.

Reply:
... so this can change, however unlikely.  We should be open to it being stale and needing a refresh.